about.html

```diff
@@ -0,0 +1,121 @@
+          <li><a href="index.html#partners">Sponsor</a></li>
+          <li><a href="#"><button>TLS Campaign</button></a></li>
+        </ul>
+        <button class="menu-button"><img src="assets/hamburger.png" alt="menu" class="mobile-menu"></button>
+        <button class="hide close"><img src="assets/close.svg" alt="close" class=""></button>
+        <div class="mobile-menu-canvas hide">
+          <div class="overlay">
+            <ul>
+              <li><a href="index.html">Home</a></li>
+              <li><a href="about.html">About</a></li>
+              <li><a href="index.html#program">Program</a></li>
+              <li><a href="index.html#program">Join</a></li>
+              <li><a href="index.html#partners">Sponsor</a></li>
+            </ul>
+          </div>
+     
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <section tabindex="0" class="headline-section">
+        <p tabindex="0" class="hello">"Hello! Twenties"</p>
+        <h1 tabindex="0">TWENTIES <br> LEADERSHIP SUMMIT 2023</h1>
+        <hr>
+        <div class="description">
+          <p>Twenties Leadership Summit is a developmental program designed to create a profound shift in the thinking and behavior of twenties. The program will prepare you to be a leader who has a moral compass, values leadership and knows first-hand what it means to live a life of purpose and impact. </p>
+        </div>
+        <address>
+        Please contact us as per Email for any further questions about TLS 2023!<br><span>tls2023@twentiessummit.com</span> 
+        </address>
+      </section>
+
+      <section class="logo-description">
+        <h2>Twenties <br> Leadership Summit 2023 Logo</h2>
+        <hr>
+        <p>The logo of TLS 2023 was decided through the logo competition from 8.February to 7.February.</p>
+        <div class="logo-container"><img src="assets/logo.png" alt="logo" class="logo"></div>
+      </section>
+
+      <section class="past-summits">
+        <h2>See the past Twenties Leadership Summits</h2>
+        <hr>
+        <p>Take a look at the last two TLS which took place in Port-Harcourt and Abuja, Nigeria</p>
+        <ul>
+          <li>
+            <img src="assets/event1.png" alt="event1">
+            <p><span class="year">2021</span><br>Twenties Leadership Summit in Port-Harcourt</p>
+          </li>
+          <li>
+            <img src="assets/event2.png" alt="event1">
+            <p><span class="year">2018</span><br>Twenties Leadership Summit in Abuja</p>
+          </li>
+          
+        </ul>
+      </section>
+
+      <section class="partners">
+        <h2>Partner</h2>
+        <hr>
+        <ul>
+          <li><img src="assets/Google.png" alt=""></li>
+          <li><img src="assets/I4G.png" alt=""></li>
+          <li><img src="assets/airbnb.png" alt=""></li>
+          <li><img src="assets/Firefox.png" alt=""></li>
+          <li><img src="assets/slack.png" alt=""></li>
+        </ul>
+      </section>
```

Comment:
Kindly add the alt attribute to your images. The alt attribute provides alternative information for an image if a user for some reason cannot view your images. 😊 ☺️